azure/template.json

```diff
@@ -319,6 +319,38 @@
       "metadata": {
       "description": "Configure Number of database server CPU"
       }
+    },
+    "subscription": {
```

Comment:
use a more descriptive name for this and `resourceGroup` parameters?
prefix appServicePlan or `monitoring` or a description of what it refers to

Reply:
done... also, i have decided to remove the resourceInstace parameter, as this is nothing but the appservicename.... So i passed the variable called "appServicePlanName" instead to the parameter to the parameter of the arm template...

Reply:
thanks..also do you need to pass the `subscriptionId` ? 
can you use `subscription().id` or `subscription().subscriptionId` or use the `subscriptionResourceId()` function inside the linked template?

Reply:
good idea..!